src/elasticsearch/addl_index_transformations/portal/__init__.py

```diff
@@ -182,6 +161,10 @@ def _map(doc, clean):
     # The recursion is usually not needed...
     # but better to do it everywhere than to miss one case.
     clean(doc)
+
+    single_valued_fields = ['donor', 'origin_sample', 'source_sample', 'rui_location']
+    multi_valued_fields = ['ancestors', 'descendants', 'immediate_ancestors', 'immediate_descendants']
```

Comment:
We were just importing this from elsewhere before. No impactful change?

Reply:
yep, no change: The outside file now doesn't do anything else now, so makes sense to inline.